src/components/MultiGestureCanvas/types.ts

```diff
@@ -3,4 +3,5 @@ type ZoomRange = {
     max: number;
 };
 
-export default ZoomRange;
+// eslint-disable-next-line import/prefer-default-export
+export type {ZoomRange};
```

Comment:
What's the reason for this change?

Reply:
I understand, but in case we have more types we just change from default export to named ones, and do a refactor, wouldn't be such a big problem to do 🙂 I would prefer using a default export and removing this error supression

Reply:
@tienifr did you forget to address this feedbakc?